components/expression_language/syntax.rst

```diff
@@ -127,9 +127,16 @@ Working with Functions
 ----------------------
 
 You can also use registered functions in the expression by using the same
-syntax as PHP and JavaScript. The ExpressionLanguage component comes with one
-function by default: ``constant()``, which will return the value of the PHP
-constant::
+syntax as PHP and JavaScript. The ExpressionLanguage component comes with the
+following functions by default:
+
+* ``constant()``
+* ``enum()``
+
+Constant function
+~~~~~~~~~~~~~~~~~
+
+This function will return the value of the PHP constant::
```

Comment:
```suggestion
This function will return the value of a PHP constant::
```

Kann you also add an example for a class with a namespace? Thanks

Reply:
Done! :slightly_smiling_face: